lib/macho/macho_file.rb

```diff
@@ -424,6 +463,16 @@ def low_fileoff
       offset
     end
 
+    # Updates the number of load commands in the raw data.
+    # @param ncmds [Fixnum] the new number of commands
+    # @return [void]
+    # @private
+    def set_ncmds(ncmds)
+      fmt = Utils.specialize_format("L=", endianness)
+      new_ncmds = [ncmds].pack(fmt)
```

Comment:
How does `num` sound? I've been using conventions like `_raw` in some places to denote the already serialized forms, so it might be confusing to use it here for the unpacked value.